nautobot/extras/forms.py

```diff
@@ -870,3 +875,30 @@ class StatusModelCSVFormMixin(CSVModelForm):
         to_field_name="slug",
         help_text="Operational status",
     )
+
+
+class GraphQLQueryForm(BootstrapMixin, forms.ModelForm):
+    slug = SlugField()
+    query = TextField()
+
+    class Meta:
+        model = GraphQLQuery
+        fields = (
+            "name",
+            "slug",
+            "query",
+        )
+
+    def clean(self):
```

Comment:
I think we should move this validation into the clean() function for GraphQLQuery object, this way the validation will be available for the UI, the API and the ORM